build.sbt

```diff
@@ -38,12 +38,12 @@ lazy val pathManager = project.in(file("path-manager"))
       "-J-XX:+PrintGCDetails",
```

Comment:
The GC option `-J-XX:+UseConcMarkSweepGC` has been deprecated and causes a fatal exception in recent Java versions.

Removing the option allows us to deploy successfully ([Commit](https://github.com/guardian/path-manager/pull/70/commits/55af041e81e60f9b290ddc941764d9c845bab5d0), [RiffRaff](https://riffraff.gutools.co.uk/deployment/view/4a5e4391-5e4a-4c17-abf6-55b680444919), [Logs](https://logs.gutools.co.uk/s/editorial-tools/app/r/s/etOCU)).

As I understand it, GC now defaults to G1, which should be fine - some comparison of the different GCs can be found [here](https://kstefanj.github.io/2023/12/13/jdk-21-the-gcs-keep-getting-better.html).

Reply:
Great point, let me amend and remove this!